src/middlewares/jwtAuth.js

```diff
@@ -0,0 +1,31 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+
+dotenv.config();
+const secret = process.env.JWT_SECRET;
+
+const verifyToken = (req, res, next) => {
```

Comment:
That function was written because the auth.middleware overwrites the `res.body.role` when it appends the role to the authenticated user's role to `res.body.role` I requested a change from the author but it wasn't effected. 
Another reason why that is there is because I the initial verify time was in the services folder and it was not exactly the shortest of codes. Again, I reached out to the author and our conclusion was to do whatever will make the app work. Then later, we said we will refactor. However, the author, along with every other person kept pushing their code and raising  PRs.

When I raised it on Monday, everyone was silent and the PO said we should do things provided the app works. So I left my code there.